docker.local/config/sc.yaml

```diff
@@ -126,6 +126,8 @@ smart_contracts:
       interest_rate: 0.0
       # interest_interval is interval to pay interests for a stake
       interest_interval: 1m
+      # min_lock_period is min lock period. Default min lock period is 3 years worth of blocks.
```

Comment:
The comment perhaps is `Default lock period is 3 years worth of blocks`

Reply:
fixed